apps/web/components/pages/task/details-section/blocks/task-progress.tsx

```diff
@@ -5,74 +5,118 @@ import TaskRow from '../components/task-row';
 import { Disclosure } from '@headlessui/react';
 import { useEffect, useState } from 'react';
 import ProfileInfoWithTime from '../components/profile-info-with-time';
-import { IEmployee } from '@app/interfaces';
 import { ChevronUpIcon } from '@heroicons/react/20/solid';
 import clsx from 'clsx';
 import {
 	useAuthenticateUser,
 	useOrganizationTeams,
-	useTeamMemberCard,
+	// useTeamMemberCard,
 	//useTaskStatistics,
 } from '@app/hooks';
 import { useTranslation } from 'lib/i18n';
-//import { secondsToTime } from '@app/helpers';
-//import { useRecoilValue } from 'recoil';
-//import { timerSecondsState } from '@app/stores';
+import { secondsToTime } from '@app/helpers';
+import { OT_Member } from '@app/interfaces';
+import { ITasksTimesheet } from '@app/interfaces/ITimer';
+
+interface ITime {
```

Comment:
@desperado1802 Plz add this interface in `ITimer.ts`, we will reuse it from there.